packages/navigation/types/Navigation.js

```diff
@@ -40,7 +40,8 @@ type MMBStackNavigatorNames =
   | 'mmb.tickets.boarding_pass'
   | 'mmb.timeline.invoice'
   | 'mmb.explore'
-  | 'DetailScreen';
+  | 'DetailScreen'
+  | 'TravelDocumentScreen';
```

Comment:
Just out of curiosity: why do we have a different naming convention here?

Reply:
Not really sure 🤷‍♂️

Reply:
I believe it would be better to stick to one, to avoid confusions in the future. What do you think?

Reply:
Well, this dotted syntax is different somehow. They are actual screens on mobile, but not on ios

Reply:
Ok then. If you think that it makes sense to have this distinction, I'll approve it.